app/models/user.rb

```diff
@@ -7,4 +7,15 @@ class User < ApplicationRecord
   has_many :collections
   has_many :manifests, -> { distinct }, through: :collections # the User's library
   has_many :annotations
+
+  after_create :create_sample_data
+
+  def create_sample_data
+    samples = JSON.parse(File.read('config/sample_data.json'))
```

Comment:
One thought I have when looking at this, is would it be worth throwing this logic into a class method on the `CreateSampleData` class (maybe something like `self.for_user(user)` that would allow this method to simply be `CreateSampleData.for_user(self)` and then the `CreateSampleData` class can have all the knowledge about where the sample data lives, how to parse it, etc.